### PR TITLE
add sentry support

### PR DESCRIPTION
**Pull request by mamantoha**

### CODE REVIEW

**Comment by mamantoha** [comment]
wait until https://github.com/Sija/raven.cr/pull/31

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,4 @@
+export KEMAL_ENV=development
 export GITHUB_USER=
 export GITHUB_KEY=
+export SENTRY_DNS=
```
Comment (Sija) [inline .env.example]: typo: should be `SENTRY_DSN`

File: shard.yml [hunk 2]
```diff
@@ -37,5 +37,9 @@ dependencies:
     github: neovintage/kemal-flash
   autolink:
     github: crystal-community/autolink.cr
+  raven:
```
Comment (Sija) [inline shard.yml]:
```yml
raven:
  github: sija/raven.cr
  branch: develop
```

File: src/shards-info.cr [hunk 3]
```diff
@@ -17,6 +19,15 @@ Kemal::Session.config do |config|
   config.secret = "my_super_secret"
 end
 
+Raven.configure do |config|
+  config.async = true
+  config.current_environment = Kemal.config.env
+  config.dsn = ENV["SENTRY_DNS"]
```
Comment (Sija) [inline src/shards-info.cr]: You shouldn't need to set `config.dsn` manually if you set `SENTRY_DSN` env variable.

**Pull request merged**